### DISCUSSION TITLE
Webgazer recalibration

### DISCUSSION THREAD

**Discussion by vatsyashh** [jspsych/jsPsych#1962] [Q&A]
I am using webgazer extension in my experiment created using jspsych. To improve the calibration results after a certain while, I am planning to recalibrate it. It's not working with calling the webgazer-calibrate variable again as it gives "Tensor is disposed" error. Please help me with how can I recalibrate and improve the accuracy. @jodeleeuw @becky-gilbert

**Comment by jodeleeuw** [comment]
Hi @vatsyashh,

Can you please share more details about the experiment timeline and how you are using the `webgazer-calibrate` plugin? I haven't run into this issue before and I'm not sure how to reproduce it from your description.
